### DISCUSSION TITLE
The 2024 NetBox Community Survey is Open!

### DISCUSSION THREAD

**Discussion by jeffgdotorg** [netbox-community/netbox#16133] [Announcements]
I’m pleased to announce that the annual NetBox community survey is now open for responses, and we’ve added a new feature: prizes! Each year, we reach out to the entire NetBox user base with a short survey to learn more about who they are, how they use NetBox, and what they would most like to see in future releases. This feedback is crucial to guiding NetBox’s long-term growth and focus.

This year’s survey is a single page of 20 questions, and should take just a few minutes to complete. The results will be published in mid-June. You can find the 2023 survey results [here](https://github.com/netbox-community/netbox/discussions/12876).

We truly appreciate your participation. This year we’re backing up that sentiment with a chance to win free stuff! We’re giving away one pair of tickets to AutoCon2, and twenty NetBox Hero t-shirts, to qualified respondents; full details after the jump.

[2024 NetBox Community Survey](https://docs.google.com/forms/d/e/1FAIpQLSeILP2BJ7pRvqQD3FBKoSvGLR6Z6Cqbf4BBNOGyeoLMsJuU6g/viewform?usp=sf_link)

**Edit**: The survey is now closed. Thanks to all who participated. We will publish the results soon.

**Comment by EHRETic** [comment]
You should do a follow-up survey for feature suggestions 😉

**Comment by opericgithub** [comment]
Where can we find the results of 2024 survey?
